### PR TITLE
Bump git platinum

### PR DESCRIPTION
**Pull request by rudolfs**
I want to use the latest and greatest.

### CODE REVIEW

File: src/vcs/git.rs [hunk 1]
```diff
@@ -757,7 +757,7 @@ impl<'a> Browser<'a> {
     /// let mut browser = Browser::new(&repo, Branch::local("master"))?;
     ///
     /// // Clamp the Browser to a particular commit
-    /// let commit = Oid::from_str("223aaf87d6ea62eef0014857640fd7c8dd0f80b5")?;
+    /// let commit = Oid::from_str("a0dd9122d33dff2a35f564d564db127152c88e02")?;
```
Comment (FintanH) [inline src/vcs/git.rs:760]: Why did this need to be done? :eyes: The reason I clamped it was so that we don't need to constantly update :smile:

**Review by FintanH — COMMENTED** [review]
We still have this bamboozling case https://github.com/radicle-dev/radicle-surf/pull/155 as well, so I'm suspecting this PR won't pass CI.

File: src/vcs/git.rs (hunk 1 above)
Comment (rudolfs) [reply src/vcs/git.rs]: Hm, you're right. I wonder why the tests failed when I tried running them for the first time after bumping `git-platinum`?

Reverted this in: 301f49fc522e81e1f771629f3156b7a892b6d2ef.

Comment (rudolfs) [reply src/vcs/git.rs]: The commit count still had to be adjusted tho'...

https://github.com/radicle-dev/radicle-surf/pull/156/files#diff-eb83d6e8d5d1ac1c789414a0e87e97d7243260cd933b4d52d908644609bdbb85R934

Comment (FintanH) [reply src/vcs/git.rs]: Makes sense :+1:

**Comment by rudolfs** [comment]
Deprecated by: https://github.com/radicle-dev/radicle-surf/pull/155.

**Pull request closed without merging**